webui/react/src/pages/TrialDetails/MultiTrialDetailsHyperparameters.settings.ts

```diff
@@ -1,50 +1,61 @@
-import { array, literal, number, string, type, undefined as undefinedType, union } from 'io-ts';
+import { array, keyof, number, string, type, undefined as undefinedType, union } from 'io-ts';
 
 import { SettingsConfig } from 'hooks/useSettings';
-import {
-  ViewType,
-  VisualizationFilters,
-} from 'pages/ExperimentDetails/ExperimentVisualization/ExperimentVisualizationFilters';
-import { Scale } from 'types';
+import { Metric, Scale } from 'types';
 
 const DEFAULT_BATCH = 0;
 const DEFAULT_BATCH_MARGIN = 10;
 const DEFAULT_MAX_TRIALS = 100;
 
 export interface ExperimentHyperparametersSettings {
-  filters: VisualizationFilters;
+  batch: number;
+  batchMargin: number;
+  hParams: string[];
+  maxTrial: number;
+  metric?: Metric;
+  scale: Scale;
```

Comment:
nice I was going to suggest flattening this in the previous commit but you already did it here